resources/views/livewire/custom-field-set-default-values-for-model.blade.php

```diff
@@ -1,54 +1,79 @@
-<span> {{-- This <span> doesn't seem to fix it, neither does a div? --}}
+<span>
+
     <div class="form-group{{ $errors->has('custom_fieldset') ? ' has-error' : '' }}">
-        <label for="custom_fieldset" class="col-md-3 control-label">{{ trans('admin/models/general.fieldset') }}</label>
-            <div class="col-md-5">
-                {{ Form::select('fieldset_id', Helper::customFieldsetList(), old('fieldset_id', $fieldset_id), array('class'=>'select2 js-fieldset-field livewire-select2', 'style'=>'width:100%; min-width:350px', 'aria-label'=>'custom_fieldset', 'data-livewire-component' => $_instance->id)) }}
-                {!! $errors->first('custom_fieldset', '<span class="alert-msg" aria-hidden="true"><br><i class="fas fa-times"></i> :message</span>') !!}
-            </div>
-            <div class="col-md-3">
-                    <label class="form-control">
-                    {{ Form::checkbox('add_default_values', 1, old('add_default_values', $add_default_values), ['data-livewire-component' => $_instance->id, 'id' => 'add_default_values', 'wire:model' => 'add_default_values']) }}
-                    {{ trans('admin/models/general.add_default_values') }}
-                </label>
-            </div>
+        <label for="custom_fieldset" class="col-md-3 control-label">
+            {{ trans('admin/models/general.fieldset') }}
+        </label>
+        <div class="col-md-5">
+            {{ Form::select('fieldset_id', Helper::customFieldsetList(), old('fieldset_id', $fieldset_id), array('class'=>'select2 js-fieldset-field livewire-select2', 'style'=>'width:100%; min-width:350px', 'aria-label'=>'custom_fieldset', 'data-livewire-component' => $_instance->id)) }}
+            {!! $errors->first('custom_fieldset', '<span class="alert-msg" aria-hidden="true"><br><i class="fas fa-times"></i> :message</span>') !!}
+        </div>
+        <div class="col-md-3">
+                <label class="form-control">
+                {{ Form::checkbox('add_default_values', 1, old('add_default_values', $add_default_values), ['data-livewire-component' => $_instance->id, 'id' => 'add_default_values', 'wire:model' => 'add_default_values']) }}
+                {{ trans('admin/models/general.add_default_values') }}
+            </label>
+        </div>
     </div>
+
     @if ($this->add_default_values ) {{-- 'if the checkbox is enabled *AND* there are more than 0 fields in the fieldsset' --}}
-    <div style="padding-left: 10px; padding-bottom: 0px; margin-bottom: -15px;">
-        <div class="form-group">
             @if ($fields)
+
                 @foreach ($fields as $field)
                     <div class="form-group">
-                    
-                            <label class="col-md-3 control-label{{ $errors->has($field->name) ? ' has-error' : '' }}" for="default-value{{ $field->id }}">{{ $field->name }}</label>
```

Comment:
I'm a _little_ worried about removing this ` for="default-valueBLAH"` attribute clause from the HTML - will this mean that we lose the ability to click on the label, and then have the appropriate element automatically come into focus? Not a dealbreaker for me, but just wanting to make sure that was a deliberate choice on your part. Especially since this `<label> ... </label>` thing doesn't wrap the `<input...>` element.

Reply:
It is deliberate, yeah - the label can't span multiple divs, which is why we use the `for` part for non-checkbox/non-radio elements.